Gems/Atom/Feature/Common/Assets/Shaders/Materials/Eye/Eye_SurfaceData.azsli

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+// This #define magic lets you use the Surface class in this file without making it the final Surface
+// used in your shader. Simply #define Surface to your custom definition before including this file
+//
+#ifndef Surface
+#define Surface     SurfaceData_Skin
+#endif
+
+#include "../BasePBR/BasePBR_SurfaceData.azsli"
+#include <Atom/Features/PBR/Surfaces/TransmissionSurfaceData.azsli>
+
+class SurfaceData_Skin
+{
+    TransmissionSurfaceData transmission;
+
+    // ------- BasePbrSurfaceData -------
```

Comment:
Since this is repeated a lot, I wonder if we should have a `BaseSurfaceData` struct somewhere

Reply:
This will be fixed with class inheritance